src/test/kotlin/io/github/mojira/arisa/modules/CrashModuleTest.kt

```diff
@@ -333,7 +333,8 @@ class CrashModuleTest : StringSpec({
             crashReader
         )
 
-        val attachment = Attachment("crash.txt", Calendar.getInstance().time) { EXAMPLE_CRASH.toByteArray() }
+        val attachment =
+            Attachment("crash.txt", Calendar.getInstance().time, { Unit.right() }, { EXAMPLE_CRASH.toByteArray() })
```

Comment:
(Minor) (Trivial) I don't like that line break here

Reply:
I'd suspect this is an automatic thing by IntelliJ. I do prefer to have parameters on seperate lines as well tho.

Reply:
It was automatically added by intellij as the line is too long 😅